experimental/packages/otlp-exporter-base/src/OTLPExporterBase.ts

```diff
@@ -110,16 +110,24 @@ export abstract class OTLPExporterBase<
     return this._shutdownOnce.call();
   }
 
+  /**
+   * Exports any pending spans in the exporter
+   */
+  forceFlush(): Promise<void> {
+    return new Promise((resolve, reject) => {
+      Promise.all(this._sendingPromises).then(() => {
```

Comment:
The return value of `Promise.all` can be returned as the result of `forceFlush`.

Reply:
hmm my IDE was showing me type errors as Promise.all has type Promise<unknown[]>, am I missing something?

Reply:
Here 3 promises were created, 1. `new Promise()`, 2. `Promise.all()`, 3. `.then()`. I think for type conformance we can cast the return value of `Promise.all` to `Promise<void>` since the return value is negligible. However, if type-casting is something you would like to avoid, `Promise.all().then(() => {})` should be fine with 2 promises created.

Either way, I believe the promise created at (1) can be removed.